app/router.js

```diff
@@ -65,12 +65,12 @@ Router.map(function() {
 
   this.route('project', { path: '/:slugged_route_slug/:project_slug' }, function() {
     this.route('checkout'); // Where you enter your credit card details
-    this.route('contributors');
-    this.route('donate'); // Where you choose your donation amount
-    this.route('messages', function() {
-      this.route('message', { path: '/:id' });
+    this.route('conversations', function() {
+      this.route('conversation', { path: '/:id' });
       this.route('new');
     });
+    this.route('donate'); // Where you choose your donation amount
+    this.route('people');
```

Comment:
I've reworked the conventions to be:

- `/contributors` => `/people`
- `/messages` => `/conversations`

The latter is more appropriate because `/messages` would actually be a different route entirely. These would be the automated/manual messages the project has sent/is actively sending to users. These would be more like `/messages/auto` and `/messages/manual`, anyway.

`/converations` serves more as a unified inbox for the project's conversations with users and replies to messages.